test/instrumentation/main_test.go

```diff
@@ -539,18 +539,6 @@ func TestIncorrectStableMetricDeclarations(t *testing.T) {
 		src      string
 		err      error
 	}{
-		{
-			testName: "Fail on stable summary metric (Summary is DEPRECATED)",
```

Comment:
Since summary's objectives are very different from histogram's buckets, could you add some tests with incorrect objective declarations in a similar fashion as to what has been done for buckets?

Reply:
We have no such tests that declare incorrect bucket declarations.

Reply:
ah, thanks for pointing that out, added a test.